public/app/features/dashboard/state/initDashboard.ts

```diff
@@ -70,7 +71,7 @@ async function fetchDashboard(
       case DashboardRoutes.Normal: {
         const dashDTO: DashboardDTO = await dashboardLoaderSrv.loadDashboard(args.urlType, args.urlSlug, args.urlUid);
 
-        if (args.fixUrl && dashDTO.meta.url) {
+        if (args.fixUrl && dashDTO.meta.url && !playlistSrv.isPlaying) {
```

Comment:
@dprokop -- I think this is what was killing the playlist cycle (old+new)

Reply:
Indeed, now it works like a charm.